app/assets/javascripts/tictactoe.js

```diff
@@ -1 +1,240 @@
 // Code your JavaScript / jQuery solution here
+
+
+var turn = 0
```

Comment:
Prefer to use `let` or `const` over `var` -- this blog post goes a bit into why: https://medium.com/javascript-scene/javascript-es6-var-let-or-const-ba58b8dcde75